src/UI/Layouts/HUD/Beta/Beta.layout

```diff
@@ -4,15 +4,15 @@ FrameWidgetClass {
   TextWidgetClass "{596565CFBE414D61}" {
    Name "URL"
    Slot FrameWidgetSlot "{596565CFBE414D45}" {
-    Anchor 0.5 1 0.5 1
-    PositionX -117.403
-    OffsetLeft -117.403
-    PositionY -42.51
-    OffsetTop -42.51
+    Anchor 0 0 0 0
+    PositionX 32.198
+    OffsetLeft 32.198
+    PositionY 10.97
+    OffsetTop 10.97
     SizeX 0
-    OffsetRight 117.403
+    OffsetRight -32.198
     SizeY 0
-    OffsetBottom 42.51
+    OffsetBottom -10.97
     SizeToContent 1
    }
```

Comment:
The URL watermark widget seems to be gone now. Please move it back :)

Reply:
It isn't it is in the top left, do you need / want it bottom center?